### PR TITLE
Renamed master to main

### PR DESCRIPTION
**Pull request by michaelgsharp**
Rename all references to machinelearning/master to machinelearning/main.

### CODE REVIEW

File: build/vsts-ci.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ resources:
   containers:
   - container: CentosContainer
     image: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-mlnet-8bba86b-20190314145033
-
+g
```
Comment (tarekgh) [inline build/vsts-ci.yml:9]: what is this "g"?

Comment (safern) [reply build/vsts-ci.yml:9]: a test for the reviewers 😆

Comment (michaelgsharp) [reply build/vsts-ci.yml:9]: Must have been an accidental keyboard hit. Good catch.

**Review by tarekgh — APPROVED** [review]

**Review by safern — APPROVED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Project Docs
 - [Strong Name Signing](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/strong-name-signing.md)
 - [Public Signing](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/public-signing.md)
```
Comment (eerhardt) [inline docs/README.md:15]: While we are in here.... can we update these to point to `dotnet/runtime`?

Comment (michaelgsharp) [reply docs/README.md]: Yep. They are now updated.

**Review by eerhardt — APPROVED** [review]

**Pull request merged**